### DISCUSSION TITLE
Poll: zkSync gas fees reduction after EIP-4844 Proto-Danksharding

### DISCUSSION THREAD

**Discussion by 0xiliyan** [zkSync-Community-Hub/zksync-developers#385] [Polls]
Proto-danksharding Ethereum Upgrade (EIP-4844) seeks to introduce a way for rollups to add cheaper data to Ethereum. This is made possible by introducing a new type of transaction that will enable blobs of data to be added to blocks.

**Comment by uF4No** [comment]
We'll some reduction but I think the available options are too optimistic. We'll see in a few weeks 😊

**Reply by 0xiliyan** [reply]
Yes the Proto-Danksharding upgrade is approaching soon so lets hope it impacts the ZK rollups like zkSync positively regarding the gas fees! (Tried adding another option (2x fee reduction) but it doesn't allow me to reorder the poll options so I didn't put it in.)

**Comment by matthewredmanv** [comment]
Proto-danksharding Ethereum Upgrade (EIP-4844) proposes a mechanism for rollups to incorporate more cost-effective data onto the Ethereum blockchain. It introduces a novel transaction type allowing for the inclusion of data blobs within blocks, enhancing scalability and efficiency.

**Comment by glasrudmary** [comment]
The proposed Proto-danksharding Ethereum Upgrade (EIP-4844) aims to enable rollups to add cheaper data to Ethereum by introducing a new type of transaction that allows for blobs of data to be added to blocks. This could potentially reduce the cost of data storage on Ethereum and make it easier for rollups to operate efficiently. However, the upgrade is still in the proposal stage and further discussion and implementation are needed to determine its feasibility and impact on the Ethereum network.
